auth.php

```diff
@@ -243,19 +243,16 @@ public function canDo($cap) {
   }
 
   public function logIn() {
-    global $QUERY;
-    $login_url = DOKU_URL . 'doku.php?id=' . $QUERY;
+    $login_url = DOKU_URL . 'doku.php?id=' . $_GET["id"];
```

Comment:
```suggestion
    global $ID;
    $login_url = wl($ID,'',true);
```

Reply:
Unfortunately, I can't test this right now, but it looks better than my solution. I will update my pull request.